### ISSUE TITLE
Abnormal circulating proteoglycan concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6000]
**Preferred term label:** Abnormal circulating proteoglycan concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** An abnormal concentration of proteoglycans in the blood., PMID: 25701227

**Parent term (use hpo.jax.org/app)** Abnormal circulating lipid concentration HP:0003119

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Abnormal circulating Lumican concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of lumican in the blood is outside the normal range., PMID: 29501701

**Parent term (use hpo.jax.org/app)** Abnormal circulating proteoglycan concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Lumican concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of lumican in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Lumican concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I do not think that we need the "Proteoglycan" grouping term, but lumican is relevant, see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4774330/

**Comment by pnrobinson** [comment]
I looked a little closer. Lumican does not have well defined normal ranges in serum. Therefore, it is not possible to say that an individual patient has an "Abnormal lumican concentration". The studies I saw relate to cohorts where the mean was higher, but that is different from an "abnormal lumican" finding.
